anvio/trnaseq.py

```diff
@@ -2068,240 +2945,1932 @@ def write_trimmed_supplement(self):
 
         self.progress.end()
 
-        self.run.info("Output trimmed tRNA file", self.trimmed_ends_path)
+        self.run.info("Trimmed tRNA supplement", self.trimmed_ends_path)
+
+
+def profile_worker(input_queue, output_queue, profiler):
+    """This client for `trnaidentifier.Profiler.profile` is located outside the `TRNASeqDataset`
+    class to allow multiprocessing."""
+    while True:
+        seq_string, seq_name = input_queue.get()
+        output_queue.put(profiler.profile(seq_string, name=seq_name))
+
+
+class NormalizedSeqSummary(object):
+    """Relevant data from normalized sequences stored in anvi'o tRNA-seq databases are reloaded into
+    these objects."""
+
+    __slots__ = (
+        'name',
+        'sample_id',
+        'seq_string',
+        'threshold_feature_start',
+        'anticodon_seq_string',
+        'feature_threshold_start',
+        'mean_specific_cov',
+        'specific_covs',
+        'nonspecific_covs',
+        'specific_nt_covs_dict',
+        'nonspecific_nt_covs_dict',
+        'mod_seq_summary'
+    )
+
+    def __init__(self):
+        self.name = None
+        self.sample_id = None
+        self.seq_string = None
+        self.threshold_feature_start = None
+        self.anticodon_seq_string = None
+        self.feature_threshold_start = None
+        self.mean_specific_cov = None
+        self.specific_covs = None
+        self.nonspecific_covs = None
+        self.specific_nt_covs_dict = None
+        self.nonspecific_nt_covs_dict = None
+        self.mod_seq_summary = None
+
+
+class ModifiedSeqSummary(object):
+    """Relevant data from modified sequences stored in anvi'o tRNA-seq databases are reloaded into
+    these objects."""
+
+    __slots__ = (
+        'name',
+        'sample_id',
+        'consensus_seq_string',
+        'sub_positions',
+        'specific_nt_covs_dict',
+        'nonspecific_nt_covs_dict',
+        'specific_covs',
+        'nonspecific_covs',
+        'specific_del_covs',
+        'nonspecific_del_covs',
+        'norm_seq_summaries'
+    )
+
+    def __init__(self):
```

Comment:
variables defined here are not introduced as members of the class with `self.`, so these lines do not contribute to anything when there is an instance as far as Python is concerned :)